automation_tools/scripts/ga-migration/templates/services/tests.yml

```diff
@@ -0,0 +1,144 @@
+name: CI
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: master
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 3 * * 6'
+  workflow_dispatch:
+    inputs:
+      reason:
+        description: 'Reason'
+        required: false
+        default: 'Manual trigger'
+
+jobs:
+  Tests:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+          # You can add/remove combinations e.g. `dev` requirements or `postgresql13` by adding
+          # a new item to the following lists.
+          # You can see the complete list of services and versions that are available at:
+          # https://docker-services-cli.readthedocs.io/en/latest/configuration.html
+          python-version: [3.6, 3.7, 3.8, 3.9]
+          requirements-level: [min, pypi]
+          {{ "cache-service: [memcached, redis]" if cache }}
+          {{ "db-service: [postgresql9, postgresql11, mysql5, mysql8]" if db }}
+          {{ "mq-service: [rabbitmq, redis]" if mq }}
+          {{ "search-service: [elasticsearch6, elasticsearch7]" if search }}
+
+          exclude:
```

Comment:
Major: we should remember to come up with the list mentioned in https://github.com/inveniosoftware/cookiecutter-invenio-module/pull/141#discussion_r534029729, to we have a list of excluded builds by default so when we make all the PRs we do not consume our quota.